adafruit_mpu6050.py

```diff
@@ -74,6 +75,29 @@
 STANDARD_GRAVITY = 9.80665
 
 
+class Clksel:  # pylint: disable=too-few-public-methods
+    """Allowed values for `clock_source` 
+
+    * :attr:'Clksel.CLKSEL_INTERNAL_8MHz
```

Comment:
I would make these :py:attr:`......` so they render properly.

Reply:
Oh I didn't see the recent change, but there's that new one, and then I would just [use pre-commit to automatically reformat your code](https://learn.adafruit.com/creating-and-sharing-a-circuitpython-library/check-your-code).  It'll handle the remaining issue the CI has with the formatting.  The CI uses `pre-commit`, so if you use it you should be all set!

Reply:
And I'll take a look after that :)

Reply:
> Oh I didn't see the recent change, but there's that new one, and then I would just [use pre-commit to automatically reformat your code](https://learn.adafruit.com/creating-and-sharing-a-circuitpython-library/check-your-code).  It'll handle the remaining issue the CI has with the formatting.  The CI uses `pre-commit`, so if you use it you should be all set!

I did use pre-commit.  Running Ubuntu 22:04.  Been making these edits on my phone because I am away from computer.   I'll re- precommit tomorrow and post a screen shot if it doesn't clear CI tonight